app/controllers/provider/admin/backend_apis_controller.rb

```diff
@@ -13,24 +13,27 @@ def index
     activate_menu :backend_apis
     search = ThreeScale::Search.new(params[:search] || params)
     @backend_apis = current_account.backend_apis
-                                            .order(updated_at: :desc)
-                                            .scope_search(search)
+                                   .order(updated_at: :desc)
+                                   .scope_search(search)
     @page_backend_apis = @backend_apis.paginate(pagination_params)
-                                          .decorate
-                                          .to_json(only: %i[name updated_at id private_endpoint system_name], methods: %i[links products_count])
+                                      .decorate
+                                      .to_json(only: %i[name updated_at id private_endpoint system_name], methods: %i[links products_count])
   end
 
   def new
     activate_menu :backend_apis
   end
 
   def create
-    if @backend_api.save
-      redirect_to provider_admin_backend_api_path(@backend_api), notice: 'Backend created'
-    else
-      flash.now[:error] = 'Backend could not be created'
-      activate_menu :backend_apis
```

Comment:
No longer needed to activate the menu in this case?

Reply:
No, since we're not refreshing the browser. The menu is already activated.

Reply:
Not even when html format is requested?